config-xml/schema/windup-jboss-ruleset.xsd

```diff
@@ -282,19 +282,15 @@
                 <xs:element ref="quickfix" minOccurs="0" maxOccurs="unbounded"/>
             </xs:sequence>
             <xs:attribute name="title" type="xs:string" use="required"/>
-            <xs:attribute name="severity" type="severity-type" use="optional" />
+            <xs:attribute name="categoryID" type="category-id-type" use="optional" />
```

Comment:
I know that is true for element names, but I wasn't sure if we wanted the same convention for attribute ids. Do you think "category-id" would be better?